components/form-control/README.md

```diff
@@ -0,0 +1,341 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+The gameface-form-control is part of the Gameface custom components suite. As most of the components in this suite, it uses slots to allow dynamic content.
+
+# Installation
+
+`npm i coherent-gameface-form-control`
+
+# Usage
+
+The gameface-form-control component comes with UMD and CJS builds.
+
+## Usage with UMD modules:
+
+* import the components library:
+
+~~~~{.html}
+<script src="./node_modules/coherent-gameface-components/umd/components.production.min.js"></script>
+~~~~
+
+* import the gameface-form-control component:
+
+~~~~{.html}
+<script src="./node_modules/gameface-form-control/umd/gameface-form-control.production.min.js"></script>
+~~~~
+
+This is all! Load the file in Gameface to see the gameface-form-control.
+
+If you wish to import the modules using JavaScript you can remove the script tags
+which import the components and the gameface-form-control from the node_modules folder and import them like this:
+
+~~~~{.js}
+import components from 'coherent-gameface-components';
+import gameface-form-control from 'gameface-form-control';
+~~~~
+
+Note that this approach requires a module bundler like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/guide/en/) to resolve the
+modules from the node_modules folder. Alternatively, you can import them directly from node_modules:
+
+~~~~{.js}
+import components from './node_modules/coherent-gameface-components/umd/components.production.min.js';
+import gameface-form-control from './node_modules/gameface-form-control/umd/gameface-form-control.production.min.js';
+~~~~
+
+## Usage with CJS modules:
+
+* Import the components library:
+
+~~~~{.js}
+const components = require('coherent-gameface-components');
+const gameface-form-control = require('gameface-form-control');
+~~~~
+
+The CommonJS(CJS) modules are used in a NodeJS environment, be sure to use a module
+bundler to use them in a browser.
+
+# Form control attributes
+
+| Attribute | Required | Values        | Default value | Usage                                                                                                                                                                            |
+| --------- | -------- | ------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `method`  | Yes      | `GET`, `POST` | N/A           | Used to specify the HTTP method that will be used to send data when the form is submitted. **If the attribute is not added or with no valid value the form won't be published.** |
```

Comment:
Why not default to GET like the standard form?

Reply:
It is .. I implemented it with GET and don't know why I leave it here with no default value